packages/flutter/lib/src/material/typography.dart

```diff
@@ -103,6 +103,39 @@ class TextTheme extends Diagnosticable {
   /// Consider using [Typography.black] or [Typography.white], which implement
   /// the typography styles in the material design specification, as a starting
   /// point.
+  ///
+  /// ## Sample code
+  ///
+  /// ```dart
+  /// /// A Widget that sets the ambient theme's title text color for its
+  /// /// descendants, while leaving other ambient theme attributes alone.
+  /// class TitleColorThemeCopy extends StatelessWidget {
+  ///   TitleColorThemeCopy({Key key, this.child, this.titleColor}) : super(key: key);
+  /// 
+  ///   final Color titleColor;
+  ///   final Widget child;
+  /// 
+  ///   @override
+  ///   Widget build(BuildContext context) {
+  ///     ThemeData theme = Theme.of(context);
+  ///     theme = theme.copyWith(
+  ///       textTheme: theme.textTheme.copyWith(
+  ///         title: theme.textTheme.title.copyWith(
+  ///           color: titleColor,
+  ///         ),
+  ///       ),
+  ///     );
+  ///     return new Theme(data: theme, child: child);
+  ///   }
+  /// }
+  /// ```
+  ///
+  /// Using [merge] instead can make more sense in some situations.
```

Comment:
This statement isn't as helpful as it should be.  Since we have both copyWith and merge, we should provide some rule of thumb here.  The see also comment appears to be headed in the right direction...

Reply:
OK, I reworked those both into descriptive See Also entries.